File: leetcode/easy/TwoSumArray.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        target_dict = {}
        for i, num in enumerate(nums):
            if num in target_dict:
                return [i, target_dict[num]]
            else:
                target_dict[target-num] = i

File: leetcode/easy/TwoSumArray.js
var twoSum = function(nums, target) {
    let map = new Map();
    for (let i=0; i<nums.length ; i++) {
        if (map.has(nums[i])) {
            return [i, map.get(nums[i])];
        } else {
            map.set(target-nums[i], i);
        }
    }
};
